src/Simplex_tree/concept/SimplexTreeOptions.h

```diff
@@ -31,5 +31,7 @@ struct SimplexTreeOptions {
   static const bool link_nodes_by_label;
   /// If true, Simplex_handle will not be invalidated after insertions or removals.
   static const bool stable_simplex_handles;
+  /// If true, assumes that Filtration_value is vector-like instead of float-like. This also assumes that Filtration_values is a class, which has a push_to method to push a filtration value $x$ onto $this>=0$. 
```

Comment:
For a better documentation rendering:
```suggestion
  /// If true, assumes that Filtration_value is vector-like instead of float-like. This also assumes that SimplexTreeOptions::Filtration_value is a class, which has a `push_to` method to push a filtration value \f$x\f$ onto \f$this>=0\f$. 
```
![image](https://github.com/GUDHI/gudhi-devel/assets/10407034/2f234aeb-6ac9-4b47-ac1b-61d8237f9ad0)